.github/workflows/check_pr.yml

```
@@ -0,0 +1,53 @@
+name: Verilator PR ci
```

```suggestion
name: Verilator PR CI
```

Sorry the mixed case bothers me ;)